website/src/theme/ReactLiveScope/index.ts

```diff
@@ -6,13 +6,17 @@
  */
 
 import React from 'react';
-import * as components from './components';
+import {createRequire} from 'react-runner';
```

Comment:
or we can just define `createRequire` it here, which is super simple
```ts
export const createRequire = (imports: Scope) => (module: string): Scope => {
  if (!imports.hasOwnProperty(module)) {
    throw new Error(`Module not found: '${module}'`)
  }
  return imports[module]
}
```

Reply:
Is there a way to encapsulate this complexity so that the user just has to provide a map of components, no extra API needed?